vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -418,6 +423,9 @@ func (d *VolumeDriver) Mount(r volume.MountRequest) volume.Response {
 func (d *VolumeDriver) Unmount(r volume.UnmountRequest) volume.Response {
 	log.WithFields(log.Fields{"name": r.Name}).Info("Unmounting Volume ")
 
+	// dirty the background refcounting process if any
+	d.refCounts.MarkDirty()
```

Comment:
Yes. We can have a separate issue for to refactor the common code between the drivers